calcom/cal.com
# 🧹 One `calcom/ui` import to rule them all

Required for:  

https://github.com/calcom/website/pull/398

https://github.com/calcom/console/pull/44

## What does this PR do?

Removes more v1 components and replaces them with the new versions. 

This PR includes changes to: 
* Removal of old EmptyScreen component. Was used in:
- [ ] License required page shown on the workflow page
- [ ] Workflow overview page, empty screen shown when you have 0 workflows.
* Removed v1 of workflow pages. They weren't used anymore. Any diff you see in these pages are because v2 files had the same name. 
- [ ] Confirm workflow pages still look okay
* Removed 3 out of 4 unused components from `/apps/web/components/integrations`. (CalendarListContainer, DisconnectIntegration, IntegrationListItem). All of them had duplicates in other places that were used instead of these.
* Removed v1 of `Switch` component and replace it with v2. Was still used at:
- [ ] API key dialog form in `/settings/developer/api-keys` modal
* Deleted v1 components of API-keys section in settings
- [ ] Confirm the modal still looks okay
* Removed v1 version of `List` component. Was still used at:
- [ ] `/getting-started` flow in the step where you connect calendars. The list of calendars is the `List` component. 

**Environment**: Staging(main branch)

## Type of change


## How should this be tested?

Use the checks in the description above to test and verify that the changes didn't break anything 🙏
